src/libcephfs.cc

```diff
@@ -2107,6 +2107,10 @@ extern "C" void LL_onfinish_release(void *release_data)
 extern "C" int64_t ceph_ll_nonblocking_readv_writev(class ceph_mount_info *cmount,
 						    struct ceph_ll_io_info *io_info)
 {
+  if (!io_info->write && io_info->zerocopy && io_info->iovcnt != 1) {
```

Comment:
Should add a comment here... Same comment as added to ceph_ll_readv_writev_buffer below